src/main/java/org/reaktivity/command/log/internal/LogCommand.java

```diff
@@ -37,33 +37,30 @@ public static void main(String[] args) throws Exception
         options.addOption(builder("t").hasArg().required(false).longOpt("type").desc("streams* | counters").build());
         options.addOption(builder("d").longOpt("directory").hasArg().desc("configuration directory").build());
         options.addOption(builder("v").longOpt("verbose").desc("verbose output").build());
-        options.addOption(builder("bc").longOpt("bufferCheck").desc("buffer capacity check").build());
+        options.addOption(builder("qd").longOpt("queue-depths").desc("display queue depths").build());
```

Comment:
Actually, we should probably fold this into the `t` `type` argument above instead.

```
options.addOption(builder("t").hasArg().required(false).longOpt("type").desc("streams* | counters | queues").build());
```